repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tags/TagDAOCassandraImpl.java

```diff
@@ -63,6 +63,30 @@ public class TagDAOCassandraImpl implements TagDAO, AutoCloseable {
     public static final String DEFAULT_HOST = "localhost";
     public static final String DATACENTER = "datacenter1";
 
+    private static final TagDAOCassandraImpl INSTANCE;
```

Comment:
nit: We can remove @Repository above if we don't want spring to manage lifecycle of this bean.